src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -416,6 +416,56 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * @}
    */
 
+  /**
+   * @name Batch element error handling
+   */
+  /**
+   * Tag dispatching structure intended for use selecting appropriate
+   * overloads for various error-handling modes.
+   *
+   * Many OpenAssetIO functions provide options as to whether errors are
+   * handled via throwing exceptions, or by returning a variant based
+   * result object.
+   *
+   * Variant policy overloads, when used in a batch context, will be
+   * exhaustive for all elements in the batch, a variant result
+   * containing either a @ref TraitsDataPtr or @ref BatchElementError
+   * will be provided for each @ref EntityReference provided as an
+   * argument to the operation.
+   *
+   * Exception polity overloads, when used in a batch context, will emit
+   * an exception at the first encountered @ref BatchElementError
+   * provided by the @ref ManagerInterface. This exception may not be in
+   * index order.
+   */
+  struct BatchElementErrorPolicyTag {
+    struct Variant {};
+    struct Exception {};
+
+    /**
+     * Static instantiations of the dispatch tags, conveniences to avoid
+     * the need to construct a new type for each method call.
+     */
+    static constexpr Variant kVariant{};
+    static constexpr Exception kException{};
+  };
+
+  /**
+   * Callback signature used for an unsuccessful operation on an
+   * element in a batch.
+   *
+   * This should be called for errors that are specific to a particular
```

Comment:
[consider][minor] My bad, but now we've duplicated the `BatchElementErrorCallback` we should really re-word the docstring in the correct voice.  E.g. here, "should" to be replaced with "will".

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/864/commits/11aab0f8461be11999b48341a041324838d79292